core/utils.ts

```diff
@@ -219,3 +219,12 @@ export function checkExcessProperties<T>(
     );
   }
 }
+
+export function throwIfTrailingSemicolonInQuery(session: Session, query: string) {
+  query = query.trim();
```

Comment:
nit: just inline this

Reply:
This was on purpose, otherwise I have to call `.trim()` twice, which seems more messy